source/install/install-kubernetes-mattermost.rst

```diff
@@ -76,6 +76,8 @@ Open a text editor and create a text file with the following details.
 
 Save the file as ``mattermost-installation.yaml``.
 
+To configure the Mattermost Installation with external database and filestore refere to `Configuring external database and filestore (Recommended)`_ before proceeding.
```

Comment:
```suggestion
To configure the Mattermost Installation with an external database and filestore refer to `Configuring external database and filestore (Recommended)`_ before proceeding.
```

There's a link missing here - is that intentional or is it a reference to another section of the document?

Reply:
Yes, it is a link to another section of the document.